sites/public/src/components/listing/ListingMap.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import { GoogleMap, Marker, useJsApiLoader } from "@react-google-maps/api"
+import { Listing } from "@bloom-housing/backend-core/types"
+
+type ListingMapProps = {
+  listing: Listing
+  googleMapsHref: string
+}
+
+const containerStyle = {
+  display: "block",
+  width: "100%",
+  height: "400px",
+}
+
+const ListingMap = (props: ListingMapProps) => {
```

Comment:
Let's call this ListingGoogleMap to not confuse with the other component named the same way

Reply:
Done.